pkg/runtime/bindstring.go

```diff
@@ -101,27 +99,9 @@ func BindStringToObject(src string, dst interface{}) error {
 			v.Set(reflect.ValueOf(parsedTime))
 			return nil
 		}
-
-		if t.ConvertibleTo(reflect.TypeOf(types.Date{})) {
```

Comment:
This code currently works on anything that's convertible to time or date, while your new code, which is a great idea, by the way, only works on things which implement Binder, so we're potentially breaking existing code here.

Could you leave in the old code as well as adding your new path? Check for Binder first, but fall back to the convertible test.

Reply:
Okay, I have updated the PR with this. Please re-review and let me know if it meets your expectations.